environments/staging/common/cloudfront.tf

```diff
@@ -110,3 +110,75 @@ resource "aws_cloudfront_origin_request_policy" "forward_all_qsa" {
     query_string_behavior = "all"
   }
 }
+
+# API Docs Cloudfront
+module "api_cdn" {
+  source = "git@github.com:trade-tariff/trade-tariff-platform-terraform-modules.git//aws/cloudfront?ref=aws/cloudfront-v1.2.1"
+
+  aliases         = ["api.${var.domain_name}"]
+  create_alias    = true
+  route53_zone_id = data.aws_route53_zone.this.id
+  comment         = "API Docs ${title(var.environment)} CDN"
+
+  enabled         = true
+  is_ipv6_enabled = true
+  price_class     = "PriceClass_100"
+
+  web_acl_id = module.waf.web_acl_id
+
+  logging_config = {
+    bucket = module.logs.s3_bucket_bucket_domain_name
+    prefix = "cloudfront/${var.environment}"
+  }
+
+  origin = {
+    api = {
+      domain_name = "api.${local.origin_domain_name}"
```

Comment:
Should this not be the S3 bucket domain name?

Reply:
Not sure tbh how this s3 bucket will be used with api, but this is the same config we deployed in development account atm.

Reply:
Yeah, I think @amberstarlight did the migration of these in the old world so we'll want the origin to be the s3 bucket for sure